### PR TITLE
Remove query consistency level enum

### PR DESCRIPTION
**Pull request by alexshtin**
**What changed?**
`enumspb.QUERY_CONSISTENCY_LEVEL` was removed from proto and code.

**Why?**
enum wasn't used and always hard coded to `STRONG`.

**How did you test it?**
Run all tests.

**Potential risks**
No risks.

### CODE REVIEW

File: service/history/historyEngine.go [hunk 1]
```diff
@@ -849,31 +847,31 @@ func (e *historyEngineImpl) QueryWorkflow(
...
-				return nil, err
-			}
-		}
-
-		if mutableStateResp.GetPreviousStartedEventId() <= 0 {
-			scope.IncCounter(metrics.QueryBeforeFirstDecisionCount)
-			return nil, ErrQueryWorkflowBeforeFirstDecision
-		}
-	}
+	// if queryConsistencyLevel == enumspb.QUERY_CONSISTENCY_LEVEL_EVENTUAL {
```
Comment (alexshtin) [inline service/history/historyEngine.go:850]: Just commented it out for now.

File: service/frontend/dcRedirectionHandler.go [hunk 2]
```diff
@@ -565,13 +564,12 @@ func (handler *DCRedirectionHandlerImpl) QueryWorkflow(
...
 			// 1. Query is meant to be fast, autoforwarding all queries will increase latency.
 			// 2. If eventual consistency was requested then the results from running out of local dc will be fine.
-			queryConsistencyLevel := enumspb.QUERY_CONSISTENCY_LEVEL_STRONG
-			if queryConsistencyLevel == enumspb.QUERY_CONSISTENCY_LEVEL_STRONG {
-				remoteClient := handler.GetRemoteFrontendClient(targetDC)
-				resp, err = remoteClient.QueryWorkflow(ctx, request)
-			} else {
-				resp, err = handler.frontendHandler.QueryWorkflow(ctx, request)
-			}
+			// if queryConsistencyLevel == enumspb.QUERY_CONSISTENCY_LEVEL_STRONG {
```
Comment (mfateev) [inline service/frontend/dcRedirectionHandler.go:567]: Any reason to keep this commented out code here?

File: service/history/historyEngine.go (hunk 1 above)
Comment (mfateev) [inline service/history/historyEngine.go:850]: remove, we have git for this

File: service/history/historyEngine.go [hunk 3]
```diff
@@ -899,11 +897,10 @@ func (e *historyEngineImpl) QueryWorkflow(
 	// matching safely, without violating the desired consistency level:
 	// 1. the namespace is not active, in this case history is immutable so a query dispatched at any time is consistent
 	// 2. the workflow is not running, whenever a workflow is not running dispatching query directly is consistent
-	// 3. the client requested eventual consistency, in this case there are no consistency requirements so dispatching directly through matching is safe
-	// 4. if there is no pending or started decision it means no events came before query arrived, so its safe to dispatch directly
+	// 3. if there is no pending or started decision it means no events came before query arrived, so its safe to dispatch directly
 	safeToDispatchDirectly := !de.IsNamespaceActive() ||
 		!mutableState.IsWorkflowExecutionRunning() ||
-		queryConsistencyLevel == enumspb.QUERY_CONSISTENCY_LEVEL_EVENTUAL ||
+		// queryConsistencyLevel == enumspb.QUERY_CONSISTENCY_LEVEL_EVENTUAL ||
```
Comment (mfateev) [inline service/history/historyEngine.go:903]: ditto

**Pull request merged**